feat(changelog): functions-added-reduce-cold-starts-with-sandbox-v2 2024-07-11 (#3467)
scaleway/docs-content: changelog/july2024/2024-07-11-functions-added-reduce-cold-starts-with-sandbox-v2.mdx

---

> title: Reduce cold-starts with sandbox V2

Reviewer: ```suggestion
title: Reduce cold starts with sandbox V2
```

status: added
author:
    fullname: 'Join the #serverless-functions channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-07-11
category: serverless
product: functions
---

> You can now select the Sandbox V2 option offering lower cold-stars for your Functions, we recommand using it for resources with more than 0.5 vCPU.

Reviewer 2: ```suggestion
You can now select the Sandbox V2 option offering lower cold-starts for your Functions. We recommend using it for resources with more than 0.5 vCPU.
```

> You can now select the Sandbox V2 option offering lower cold-starts for your Functions. We recommend using it for resources with more than 0.5 vCPU.

Reviewer: ```suggestion
You can now select the Sandbox V2 option offering shorter cold starts for your Functions. We recommend using it for resources with more than 0.5 vCPU.
```
